### PR TITLE
chore: Upgrade nearcore to 1.32.2

### PR DESCRIPTION
**Pull request by WuTao18**

### CODE REVIEW

File: src/models/serializers.rs [hunk 1]
```diff
@@ -130,6 +130,7 @@ pub(crate) fn extract_action_type_and_value_from_action_view(
                 "beneficiary_id": beneficiary_id,
             }),
         ),
+        ActionView::Delegate { .. } => (ActionKind::DelegateAction, json!({})),
```
Comment (khorolets) [inline src/models/serializers.rs:133]: This is a missing implementation part.

**Review by khorolets — CHANGES_REQUESTED** [review]
Hey there! Thank you for your kind contribution to keeping the `nearcore`-based version of Indexer for Explorer alive.

However, we cannot accept the PR with a missing implementation. You've left a "placeholder" for the new `DelegateAction` instead of the real implementation. 

If you struggle with how to implement you can have a look at the related NEP-366 support PR we've merged into the current Lake-based version of the project here https://github.com/near/near-indexer-for-explorer/pull/361

File: src/models/serializers.rs (hunk 1 above)
Comment (WuTao18) [reply src/models/serializers.rs:133]: Thank you for providing the information, I will try to implement this part.

Comment (khorolets) [reply src/models/serializers.rs:133]: Thanks! It means a lot, appreciate you helping the community of developers using the version we cannot support anymore! ❤️

**Review by khorolets — APPROVED** [review]
At a glance, the PR looks legit. However, I don't have the bandwidth to test it, I hope @WuTao18 had it running and can confirm it's working as expected. 

I am approving it, @WuTao18 feel free to merge if you're confident enough.

**Comment by WuTao18** [comment]
I've been running it for a week now, everything is working fine so far. The data related to `DelegateAction` can be inserted into the PostgreSQL database. I also manually selected some data and compared it with the public database (`postgres://public_readonly:nearprotocol@mainnet.db.explorer.indexer.near.dev/mainnet_explorer`), and the data is consistent.

**Comment by khorolets** [comment]
> I've been running it for a week now, everything is working fine so far. The data related to `DelegateAction` can be inserted into the PostgreSQL database. I also manually selected some data and compared it with the public database (`postgres://public_readonly:nearprotocol@mainnet.db.explorer.indexer.near.dev/mainnet_explorer`), and the data is consistent.

Thank you for confirming, you're free to merge it. Great job!

**Comment by WuTao18** [comment]
> Thank you for confirming, you're free to merge it. Great job!

Sorry, I don’t have write access, so I can’t merge this pr. @khorolets  Could you please merge this pr? Many thanks!

**Pull request merged**